docs/agent_container_manual_deploy.md

```diff
@@ -0,0 +1,76 @@
+## Instructions for starting an agent in a container on Linux
+These instructions assume that the agent container you wish to deploy already exists and is in a container image repository somewhere.
+
+### Prerequisites
+Docker needs to be installed on the host device. For instructions on installing Docker on Linux see [here](https://docs.docker.com/engine/install/).
+
+Need to have the SSL certificate from the management hub.
+
+### Starting the agent
+Before starting the container, create a config file for the agent. This should include the following:
+ * HZN_EXCHANGE_URL=<adress of your exchange>
+ * HZN_FSS_CSSURL=<css address>
+ * HZN_ORG_ID=<the org this node should be in>
+ * HZN_EXCHANGE_USER_AUTH=<exchange username:password>
+Optional:
+ * HZN_DEVICE_ID=<a name for your node>
```

Comment:
device id is optional, but if they dont use it, I think they will get a long generated number as the node id.

Reply:
Added a note about this